platform/projects_and_organizations/members.mdx

```diff
@@ -24,6 +24,7 @@ You can choose the permissions that you want to give to the member. The permissi
 - **Translate** – The member can change the translations in the selected language(s).
 - **Review** – The member can change the translations and mark them as reviewed in the selected language(s).
 - **Edit** – The member can change the translations, mark them as reviewed, and add new keys.
+  The member can also view all recent batch operations in the Editor and start the "Pre-translate by TM" and "Machine translation" batch operations.

```

Comment:
Do we use the term "Editor" in our documentation. Because in the app we don't, so we should probably switch the terms in both app in docs or we shall stay with the "Translations" term.